rsts/user/features/fast_registration.rst

```diff
@@ -9,10 +9,67 @@ Fast Registration
 Are you frustrated by having to wait for an image build in order to test out simple code changes to your Flyte workflows? If you're interested in reducing to your iteration cycle to mere seconds, read on below.
 
 Caveats
-#######
+=======
 
 Fast-registration only works when you're testing out code changes. If you need to update your container, say by installing a dependency or modifying a Dockerfile, you **must** use the conventional method of committing your changes and rebuilding a container image.
 
+Prerequisites
+=============
+
+* Upgrade your flytekit dependency to ``>=0.16.0`` and re-run piptools if necessary
+
+
+You'll need to build a base image with these changes incorporated before you can use fast registration.
+
+
+Fast-registering
+================
+
+How-to:
+
+* After the above prerequisite changes are merged, pull the latest master and create a development branch on your local machine.
+* Make some code changes. Save your files.
+* Clear and/or create the directory used to store your serialized code archive:
+
+.. code-block:: text 
+
+   mkdir _pb_output || true
+   rm -f _pb_output/*.tar.gz 
+
+* Using a python environment with flytekit installed fast-register your changes:
+
+.. code-block:: python 
+
+   pyflyte -c sandbox.config --pkgs recipes serialize --in-container-config-path /root/sandbox.config \
```

Comment:
sorry remind me again, these commands work for both new and old style tasks/classes?

Reply:
they should since presumably we didn't break serialization 🤞